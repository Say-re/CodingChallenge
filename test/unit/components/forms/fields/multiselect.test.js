import React from 'react';
import { shallow } from 'enzyme';
import MultiSelectComponent from '../../../../../src/components/forms/fields/multiSelect'

describe('Multiselect Component Tests', () => {
  test('MultiSelectComponent Snapshot Test', () => {
    const wrapper = shallow(<MultiSelectComponent />)
    expect(wrapper).toMatchSnapshot();
  })
})
