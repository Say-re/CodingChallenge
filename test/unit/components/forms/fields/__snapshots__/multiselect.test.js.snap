// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Multiselect Component Tests MultiSelectComponent Snapshot Test 1`] = `
<div
  id="multiselectWrapper"
>
  <label
    for="codingExperience"
  >
    Please Select Coding Experience - select all that apply
  </label>
  <select
    id="codingExperience"
    multiple={true}
    onChange={[Function]}
    size={10}
  >
    <option
      value="Javascript"
    >
      Javascript
    </option>
    <option
      value="HTML"
    >
      HTML
    </option>
    <option
      value="CSS"
    >
      CSS
    </option>
    <option
      value="SASS"
    >
      SASS
    </option>
    <option
      value="NodeJS"
    >
      NodeJS
    </option>
    <option
      value="Java"
    >
      Java
    </option>
    <option
      value="AWS - Lambda"
    >
      AWS - Lambda
    </option>
    <option
      value="AWS - IAM"
    >
      AWS - IAM
    </option>
    <option
      value="AWS - DynamoDB"
    >
      AWS - DynamoDB
    </option>
    <option
      value="S3"
    >
      S3
    </option>
  </select>
</div>
`;
